app/dashboard/templates/shared/onepager_auth.html

```diff
@@ -36,11 +36,11 @@
   </div>
   
   {% if user.is_authenticated %}
-  <div class="nav-link login">
+  <a class="nav-link login" href="/profile/">
```

Comment:
Can you modify this to use the django template g engine via `{% url "profile" %}`